### ISSUE TITLE
Unable to comment

### ISSUE THREAD

**Issue by unusualbob** [martindale/converse#11]
```
code: 17280
err: "Btree::insert: key too large to index, failing converse.comments.$hashcash_1 5239 { : "2:2:sha256:20150715:004f006e00650020006f006600200074006800650020006d006f0072006500200063006f006d006d006f006e0020006300680061006e0067006500730020006100..." }"
name: "MongoError"
```

Looks like you are trying to unique index on a very large text field? Mongo will die if you try that, you may be better off making a smaller length hash of the work and making that a unique index.

**Comment by martindale** [comment]
Ah, inadvertently created an index with `sparse`.  Fixed in d9c5043.

**Comment by unusualbob** [comment]
This appears to work properly now
